pkg/controllers/nodeclaim/termination/controller.go

```diff
@@ -89,14 +95,26 @@ func (c *Controller) finalize(ctx context.Context, nodeClaim *v1beta1.NodeClaim)
 			}
 		}
 	}
-	// We wait until all the nodes associated with this nodeClaim have completed their deletion before triggering the finalization of the nodeClaim
```

Comment:
Why are we dropping waiting for the nodes to terminate in the NodeClaim termination?

Reply:
Without this here, this is going to cause us to call instance termination before the nodes are fully drained